config.toml

```diff
@@ -19,13 +19,13 @@ url = "/channels/"
 weight = 2
 
 [[ menu.main ]]
-name = "Playlists"
-url = "/playlists/"
+name = "Videos"
+url = "/videos/"
 weight = 3
 
 [[ menu.main ]]
-name = "Blog"
```

Comment:
Going to keep the blog dirs and data around for later?

Reply:
Yeah still gonna use them, they’re just linked in the footer instead